src/App/Header/index.tsx

```diff
@@ -35,27 +36,42 @@ export default function Header({ classes, links, logo }: Props) {
   const mb = hasBanner(location) ? "-mb-[6.5rem]" : "mb-0";
 
   return (
-    <header
-      className={`${classes} ${isScrolled ? "shadow-lg" : ""} ${bg} ${mb}
+    <>
```

Comment:
@SovereignAndrey, the overall page layout is 
```
grid auto_1fr_auto
header
-
-
-
page
-
-
-
footer
```
with the header now being two elements enclosed by `<></>

the layout becomes

```
grid auto_1fr_auto
notice
header
-
-
-
body
-
-
-
footer
```
causing other pages to break
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/97cc69f7-cac2-4e45-8246-92a7434aab05)


Solution would be to change `src/App/Layout.tsx` grid layout to `auto_auto_1fr_auto` to account for the notice in inside the `<Header/>` 

this change however would make such layout not obvious
as it is expected that the child is `4` elements, to make it more evident
it's probably best to just put `Notice` in `src/App/Layout.tsx`
`i.e.`

```
div grid auto_auto_1fr_auto
  --Seo (not rendered)
  --{...insert notice here..} (auto)
  --Header (auto)
  --Body (1fr)
  Footer (auto)
 div
```

Reply:
@ap-justin I hate to make big layout changes for a hotfix that will only exist for 3 weeks before being removed. :thinking: What if we only render that on the marketplace page? `if route == marketplace ? show : return nothing`